stock_cycle_count/models/stock_move.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.multi
+    def action_done(self):
+        super(StockMove, self).action_done()
+        for move in self:
+            move.location_id.check_zero_confirmation()
```

Comment:
check_zero_confirmation() function is a real multi, maybe it could be called on all locations ?